### ISSUE TITLE
[US_ND] P1 bugs for ND launch

### ISSUE THREAD

**Issue by jovergaag** [Recidiviz/pulse-dashboard#1614]
**What needs to be done? Why does it need to be done?**
These should be completed only after the P0 bugs have been finished!

- [x] List of people has a strange white box at the bottom of the table on small screen widths.
![Screen Shot 2022-01-27 at 3.45.45 PM.png](https://images.zenhubusercontent.com/5f68f900f5589b314e6a6ec3/36f9a06e-6de4-4736-98f7-20cde105270b)

- [x] add padding between rows of filters when filters wrap to a second row
![Screen Shot 2022-01-27 at 3.41.58 PM.png](https://images.zenhubusercontent.com/5f68f900f5589b314e6a6ec3/6e5461ae-afe3-44e4-9a2f-1197b4b1d873)

- [x] x-axis labels are squished on the over time charts at the 5 year time period on smaller screens. Let's reduce the number of labels rendered.!![Screen Shot 2022-01-27 at 3.42.53 PM.png](https://images.zenhubusercontent.com/5f68f900f5589b314e6a6ec3/0dccf750-8567-4962-91ad-9e45d59ea3af)

- [x] There are reports of a horizontal scroll bar on the person level table no matter the screen width. I do not see this on my browser, maybe it is a browser setting? Something to look into. We don't want the horizontal scroll on this table unless in tablet width

- [x] On narrow screen widths the age label on the age charts is squished. Let's change these labels to "<25", "25-29" etc and keep the full label in the tooltip hover. The facilities chart is doing something similar.

P2 (save for last)
- [x] Add padding to the bottom below the last section navigation item. Its too hard to click on the "race" section on smaller screen sizes
![Screen Shot 2022-01-27 at 3.43.56 PM.png](https://images.zenhubusercontent.com/5f68f900f5589b314e6a6ec3/6a23b86f-1ec7-483f-8241-18ac47b007bd)

- [x] At the right screen width, the "..." button wraps on its own line - and the modal that opens when it is clicked is in the wrong place. Let's anchor the "..." button in place at the right side and have the filters wrap instead of the "..."

**Additional context**
Add any other context about the task here.

**Comment by jovergaag** [comment]
@nasaownsky I tagged you in a new issue about adding a `longLabel` key to the `filterOptions`. That might be helpful with the age group labels being squished. That change came out of a conversation with the designer this morning, sorry if the requirements didn't make it to you before you started working on that change!

**Comment by jovergaag** [comment]
I'm also going to add another P1 here - The "race" labels should keep the same copy, but please word wrap them and angle the labels on the chart.

![Screen Shot 2022-01-31 at 1 20 45 PM](https://user-images.githubusercontent.com/5872651/151875084-0659d4f2-f598-4ba4-aee7-9d9150233b18.png)
